### PR TITLE
readme: fix typo for Atk4\Api link; remove outdated links

### PR DESCRIPTION
**Pull request by PhilippGrashoff**
- There were some links not reachable or the browser complaining that its not a secure connection, they were removed
- Add a link to our discord channel instead
- fix link text for Atk4\Api

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 [Agile Toolkit](https://atk4.org/) is a Low Code framework written in PHP. Agile UI implement server side rendering engine and over 50 UI generic components for interacting with your Data Model.
 
-Agile Data is a framework for defining your "business layer" which is separate from your "presentation layer" and "persistence". Together with [Agile UI](https://github.com/atk4/ui) you can deliver user interface "out of the box" or with [Agile UI](https://github.com/atk4/api) - general-purpose API endpoints.
+Agile Data is a framework for defining your "business layer" which is separate from your "presentation layer" and "persistence". Together with [Agile UI](https://github.com/atk4/ui) you can deliver user interface "out of the box" or with [Atk4\Api](https://github.com/atk4/api) - general-purpose API endpoints.
```
Comment (mvorisek) [inline README.md]: I completely agree to use one name. In most of the places, we use `atk4/xxx` (in LC, and forward slash). What about converting every "Agile Xxx" to `atk4/xxx`?

File: README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Agile Data is a framework for defining your "business layer" which is separate f
 [![GitHub release](https://img.shields.io/github/release/atk4/data.svg)](CHANGELOG.md)
 [![Code Climate](https://codeclimate.com/github/atk4/data/badges/gpa.svg)](https://codeclimate.com/github/atk4/data)
 
-Quick-Links: [Documentation](https://atk4-data.readthedocs.io). [Example](https://github.com/atk4/data-primer). [ATK UI](https://github.com/atk4/ui). [Forum](https://forum.agiletoolkit.org/). [Commercial support](https://www.agiletoolkit.org/contact). [Udemy Course](https://www.udemy.com/web-apps-with-php-and-atk/).
+Quick-Links: [Documentation](https://atk4-data.readthedocs.io) | [Discord channel](https://discord.com/channels/657649422280425523) | [ATK UI](https://github.com/atk4/ui)
```
Comment (mvorisek) [inline README.md]: IIRC, the Discord link cannot be opened by people that have not joined the channel yet.

see alternative in https://github.com/atk4/ui/commit/6ef7ad6ea54d66fc0dcd176a9418b2c3c6404b94

File: README.md (hunk 1 above)
Comment (PhilippGrashoff) [reply README.md]: Hi, I replaced it with atk4/api. I agree that we should have a clear naming. This PR was just a quick fix - these outdated links will scare away any new developer checking the repo. When I have more time, I can do more work on the readme like renaming, updated code samples and also some restructuring.

File: README.md (hunk 2 above)
Comment (PhilippGrashoff) [reply README.md]: changed, thx

**Comment by mvorisek** [comment]
The issue was closed as you synced the develop branch... Better to never open a PR with your forked develop.

https://github.com/atk4/data/compare/a53e6efdedf0dad78fc462fe21fec08821a17385 changes are welcomed and I will push them as part of another PR, as the changes are small.

**Comment by PhilippGrashoff** [comment]
Yes, beginners fault with pushing changes directly to develop :) I will create a new PR with exactly the code which I added to a separate branch.

**Pull request closed without merging**